static/privacy.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<style>
+
+</style>
+<body>
+<h2>What do we collect? How is it used?</h2>
```

Comment:
I have a ton of notes on what will go here. I'll update this once I'm happy with them